src/Features.php

```diff
@@ -66,6 +70,24 @@ private static function get_fields(): array {
 				'id'   => self::CLOUDFLARE_DEPLOY_PURGE,
 				'type' => 'checkbox',
 			],
+			[
+				'name' => __( 'Theme editor (experimental)', 'planet4-master-theme-backend' ),
+				'desc' => __(
+					'Enable CSS variables based theme editor for logged in users. Do not use in production yet.',
+					'planet4-master-theme-backend'
+				),
+				'id'   => self::THEME_EDITOR,
+				'type' => 'checkbox',
+			],
+			[
+				'name' => __( 'Theme editor non-logged in(experimental)', 'planet4-master-theme-backend' ),
```

Comment:
I made this a separate checkbox to be sure people see the warning about enabling it for all users.

Reply:
I left a comment on the other PR. I think it would make more sense not to enable that for non-logged-in users.